src/dev/ci_setup/setup_env.sh

```diff
@@ -71,7 +73,7 @@ if [[ "$OS" == "win" ]]; then
   nodeUrl="https://us-central1-elastic-kibana-184716.cloudfunctions.net/kibana-ci-proxy-cache/dist/v$nodeVersion/node-v$nodeVersion-win-x64.zip"
 else
   nodeBin="$nodeDir/bin"
-  nodeUrl="https://us-central1-elastic-kibana-184716.cloudfunctions.net/kibana-ci-proxy-cache/dist/v$nodeVersion/node-v$nodeVersion-linux-x64.tar.gz"
+  nodeUrl="https://us-central1-elastic-kibana-184716.cloudfunctions.net/kibana-ci-proxy-cache/dist/v$nodeVersion/node-v$nodeVersion-${OS}-x64.tar.gz"
```

Comment:
Since the URL will also work with Windows, mind moving the `nodeUrl` outside the condition block so we only have a single place where `nodeUrl` is set?

Reply:
See https://github.com/elastic/kibana/pull/51705/commits/003a36103d7c12c7d971bf2c47f28bf39a0a3a7a